src/arrow-sdk.ts

```diff
@@ -442,18 +449,20 @@ export async function prepareDeliverOptionParams(
     let strikeType = undefined
 
     switch(version) {
-        case VERSION.V2: {
+        case VERSION.V2: 
             formattedStrike = (optionOrderParams.strike as number).toFixed(2)
             bigNumberStrike = ethers.utils.parseUnits(formattedStrike, stablecoinDecimals)
             strikeType = 'uint256'
-        }
+            break;
         case VERSION.V3:
-        case VERSION.COMPETITION: {
+        case VERSION.COMPETITION: 
             const strikes = (optionOrderParams.strike as number[]).map(strike => strike.toFixed(2))
             bigNumberStrike = strikes.map(strike => ethers.utils.parseUnits(strike, stablecoinDecimals))
             formattedStrike = strikes.join('|')
             strikeType = 'uint256[2]'
-        }
+            break;
+        default:
+            throw UNSUPPORTED_VERSION_ERROR;
```

Comment:
For consistency, can you remove the semicolons at the end of lines that you have added? Sorry about that